src/utilities/filetypes/EpwFile.cpp

```diff
@@ -3887,6 +3887,17 @@ bool EpwFile::parse(std::istream& ifs, bool storeData) {
     return false;
   }
 
+  struct epw_string
+  {
+    int lineNumber;
```

Comment:
Perhaps these can be const based on your use case.

Reply:
Changed `int` to `const int`.